ui/components/component-library/container/container.types.ts

```diff
@@ -0,0 +1,28 @@
+import type {
+  StyleUtilityProps,
+  PolymorphicComponentPropWithRef,
+} from '../box';
+
+export enum ContainerMaxWidth {
+  Sm = 'sm',
+  Md = 'md',
+  Lg = 'lg',
+}
+
+export interface ContainerStyleUtilityProps extends StyleUtilityProps {
+  /*
+   * Additional classNames to be added to the Container component
+   */
+  className?: string;
+  /**
+   * maxWidth prop sets the max-width of the Container component
```

Comment:
Do we want to add the max width sizes here for intellisense? 

```suggestion
   * maxWidth prop sets the max-width of the Container component
   * Sm (320px)
   * Md (480px)
   * Lg (720px)
```